app/models/condition.rb

```diff
@@ -3,7 +3,7 @@ class Condition < ApplicationRecord
   before_validation :default_name_to_guid, :on => :create
 
   validates :name, :description, :guid, :expression, :towhat, :presence => true
```

Comment:
`:guid` can be removed from this presence check since that is handled in the mixin.